packages/now-cli/src/util/projects/link.ts

```diff
@@ -141,13 +141,25 @@ export async function getLinkedProject(
   }
 
   const spinner = output.spinner('Retrieving project…', 1000);
-  let org: Org | null;
-  let project: Project | ProjectNotFound | null;
+  let org: Org | null = null;
+  let project: Project | ProjectNotFound | null = null;
   try {
     [org, project] = await Promise.all([
       getOrgById(client, link.orgId),
       getProjectByIdOrName(client, link.projectId, link.orgId),
     ]);
+  } catch (error) {
+    if (error instanceof ProjectUnauthorized) {
+      output.error(
```

Comment:
I think we want to call `spinner()` _before_ the `output.error()` otherwise part of the message may be cut off 🤔